src/app/login/services/feedback.service.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+
+
+import { environment } from '../../../environments/environment';
+import { FeedbackService } from './feedback.service';
+import { Feedback, FeedbackContext } from '../models/feedback.model';
+
+describe('FeedbackService', () => {
+  let httpClient: HttpClient;
+  let httpTestingController: HttpTestingController;
+  let feedbackService: FeedbackService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [FeedbackService],
+      imports: [HttpClientTestingModule]
+    });
+
+    httpClient = TestBed.inject(HttpClient);
+    httpTestingController = TestBed.inject(HttpTestingController);
+    feedbackService = TestBed.inject(FeedbackService);
+  });
+
+  afterEach(() => {
+    httpTestingController.verify();
+  });
+
+  it('should be created', () => {
+    expect(feedbackService).toBeTruthy();
+  });
+
+  it('should return 201 success', () => {
```

Comment:
IMHO we only need one test here right now.  To me it's not important to test what status comes back in the response (because what status comes back is defined in the test on the mock) but what I find important is that we check that `sendFeedback` triggers a POST to `.../api/v1/feedback/login`. I would leave this test in and remove the other two.